common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -889,6 +894,7 @@ def test_submit_problem_error_with_codejail_exception(self):
 
             # Expect that the number of attempts is NOT incremented
             self.assertEqual(module.attempts, 1)
+            self.assertEqual(module.lcp.context['attempt'], 1)
```

Comment:
This should be 2, not 1. What's happening is that the student is submitting attempt #2, and the grader needs to know that this is attempt number 2, not attempt number 1.

* `module.attempts` points to the number of attempts USED
* `module.lcp.context['attempt']` points to the attempt number that is ABOUT to be used (or was just ATTEMPTED to be used)